diff --git a/sshcrypto_kex.c b/sshcrypto_kex.c
index b970fd2..f1f0202 100644
--- a/sshcrypto_kex.c
+++ b/sshcrypto_kex.c
@@ -39,18 +39,6 @@ struct sshcrypto_kex sshcrypto_kexs[] = {
         sshcrypto_TYPENEWCRYPTO,
         0,
     },
-    {
-        "sntrup761x25519-sha512",
-        crypto_kem_sntrup761x25519_enc,
-        crypto_kem_sntrup761x25519_PUBLICKEYBYTES,  /* pk */
-        crypto_kem_sntrup761x25519_CIPHERTEXTBYTES, /* c */
-        crypto_kem_sntrup761x25519_BYTES,           /* k */
-        crypto_hash_sha512,
-        crypto_hash_sha512_BYTES,
-        sntrup761x25519_putkemkey,
-        sshcrypto_TYPEPQCRYPTO,
-        0,
-    },
     {
         "sntrup761x25519-sha512@openssh.com",
         crypto_kem_sntrup761x25519_enc,
